modules/updateTask.js

```diff
@@ -0,0 +1,15 @@
+import { getTaskFromLocalStorage, setTaskIntoLocalStorage } from './localStorage.js';
+
+const updateTask = (e) => {
+  const tasks = getTaskFromLocalStorage();
+  const selectedID = parseInt(e.id, 10);
+
+  for (let i = 0; i < tasks.length; i += 1) {
+    if (tasks[i].index === selectedID) {
+      tasks[i].description = e.value;
+      break;
+    }
+  }
+  setTaskIntoLocalStorage(tasks);
+};
```

Comment:
- [ ] I cannot edit a task, if I click on task description, bin icon gets aligned to the left and task description disappears making it impossible to update it. Please make sure that users can update the task description:

https://user-images.githubusercontent.com/77038610/160114179-d4c20291-e251-47e8-9bda-8f64153184f9.mp4